### ISSUE TITLE
An ADA issue: reset password confirmation message is not read by a screen reader automatically. The screen reader starts reading the page from the top instead. This result in a bad experience to a screen reader user.

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#31]
[upwork.com/ab/f/contracts/20874286](https://www.upwork.com/ab/f/contracts/20874286)

![01](https://user-images.githubusercontent.com/83204/47613418-4ec2e300-dab9-11e8-934d-9404f99009c3.png)

## Rationale
>Screen reader should announce the error message automatically or else a user with visual
impairment who relies on screen reader for web access will need to navigate the page manually
using arrow keys in order to check if there is an error occurred while submission of the form.

## Recommendations
>Using the `aria-live` attribute on an active error message informs a screen reader user of the
change. 
`aria-live` can be placed on hidden html elements and when they are revealed via
JavaScript they will be announced to the user. 
`aria-live` will accept the values: `off`, `assertive` and `polite`. 
Using the `polite` setting, the content will only be read once the user is idle and will not
interrupt a current action. 
The `assertive` setting will be read immediately, and `off` is the default.
Using the `polite` setting is generally preferable. 
This approach is a quick and easy way to make the form’s errors accessible. 
See recommendation from W3C on how to use ARIA `role=alert` to identify errors.

## Technology
[developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)

**Comment by dmitrii-fediuk** [comment]
Done:
![02](https://user-images.githubusercontent.com/83204/47613817-bed46780-dabf-11e8-8bf5-b2a12e1dc4c9.png)
